index.bs

```diff
@@ -5525,9 +5650,11 @@ and passes them to the authenticator in the {{CredentialsContainer/create()}} ca
 {{CredentialsContainer/get()}} call (for [=authentication extensions=]). These [=authenticator extension input=] values are
 represented in [=CBOR=] and passed as name-value pairs, with the [=extension identifier=] as the name, and the corresponding
 [=authenticator extension input=] as the value. The authenticator, in turn, performs additional processing for the extensions
-that it supports, and returns the [=CBOR=] [=authenticator extension output=] for each as specified by the extension. Part of
-the [=client extension processing=] for [=authenticator extensions=] is to use the [=authenticator extension output=] as an
-input to creating the [=client extension output=].
+that it supports, and returns the [=CBOR=] [=authenticator extension output=] for each as specified by the extension.
+Since [=authenticator extension output=] is returned as part of the signed [=authenticator data=], authenticator extensions
+may also specify an [=unsigned extension output=], e.g. for cases where an output itself depends on [=authenticator data=].
```

Comment:
I agree.  And that requires appropriate changes in FIDO CTAP - since the authenticator communication is defined there (requiring an unsigned extension collection in authenticatorGetAssertion response structure and similarly for the authenticatorCreateCredential response structure). See https://github.com/fido-alliance/fido-2-specs/issues/1342